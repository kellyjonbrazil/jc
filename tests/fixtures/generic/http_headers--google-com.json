[{"_type":"response","_response_version":"HTTP/1.1","_response_status":301,"_response_reason":["Moved Permanently"],"location":"http://www.google.com/","content-type":"text/html; charset=UTF-8","content-security-policy-report-only":["object-src 'none';base-uri 'self';script-src 'nonce-hz60SAtf4llI3wIzHRfCJQ' 'strict-dynamic' 'report-sample' 'unsafe-eval' 'unsafe-inline' https: http:;report-uri https://csp.withgoogle.com/csp/gws/other-hp"],"date":"Sun, 04 Feb 2024 22:48:38 GMT","expires":"Tue, 05 Mar 2024 22:48:38 GMT","cache-control":["public","max-age=2592000"],"server":["gws"],"content-length":219,"x-xss-protection":"0","x-frame-options":"SAMEORIGIN","date_epoch_utc":1707086918,"expires_epoch_utc":1709678918},{"_type":"response","_response_version":"HTTP/1.1","_response_status":200,"_response_reason":["OK"],"content-type":"text/html; charset=ISO-8859-1","content-security-policy-report-only":["object-src 'none';base-uri 'self';script-src 'nonce-mX7dt-8o5qVlNVsgmeavew' 'strict-dynamic' 'report-sample' 'unsafe-eval' 'unsafe-inline' https: http:;report-uri https://csp.withgoogle.com/csp/gws/other-hp"],"p3p":"CP=\"This is not a P3P policy! See g.co/p3phelp for more info.\"","date":"Sun, 04 Feb 2024 22:48:38 GMT","server":["gws"],"x-xss-protection":"0","x-frame-options":"SAMEORIGIN","transfer-encoding":["chunked"],"expires":"Sun, 04 Feb 2024 22:48:38 GMT","cache-control":["private"],"set-cookie":["1P_JAR=2024-02-04-22; expires=Tue, 05-Mar-2024 22:48:38 GMT; path=/; domain=.google.com; Secure","AEC=Ae3NU9OU79owVo5-35hp-3J9oKndh-rfsAa-zY1qEjSYgasatkOvi9Cc_P8; expires=Fri, 02-Aug-2024 22:48:38 GMT; path=/; domain=.google.com; Secure; HttpOnly; SameSite=lax","NID=511=YT39CFpNgnJnUOS1bI2BM3RWycjJIJ9HfoYYzzj48nu9tym-Ja_uLvmg-vBYyJYoA7_BBrlj4nGil9o0WYbz2tfMwDVzpHXRVZTZx7P9yI-Ad9t8VgRB4wVS5nGEi9xZ20djOLr-bTWoIiJuXgUdKH30znpJXziu542KaHu0ZcQ; expires=Mon, 05-Aug-2024 22:48:38 GMT; path=/; domain=.google.com; HttpOnly"],"date_epoch_utc":1707086918,"expires_epoch_utc":1707086918}]
