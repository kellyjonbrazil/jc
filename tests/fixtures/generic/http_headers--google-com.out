HTTP/1.1 301 Moved Permanently
Location: http://www.google.com/
Content-Type: text/html; charset=UTF-8
Content-Security-Policy-Report-Only: object-src 'none';base-uri 'self';script-src 'nonce-hz60SAtf4llI3wIzHRfCJQ' 'strict-dynamic' 'report-sample' 'unsafe-eval' 'unsafe-inline' https: http:;report-uri https://csp.withgoogle.com/csp/gws/other-hp
Date: Sun, 04 Feb 2024 22:48:38 GMT
Expires: Tue, 05 Mar 2024 22:48:38 GMT
Cache-Control: public, max-age=2592000
Server: gws
Content-Length: 219
X-XSS-Protection: 0
X-Frame-Options: SAMEORIGIN

HTTP/1.1 200 OK
Content-Type: text/html; charset=ISO-8859-1
Content-Security-Policy-Report-Only: object-src 'none';base-uri 'self';script-src 'nonce-mX7dt-8o5qVlNVsgmeavew' 'strict-dynamic' 'report-sample' 'unsafe-eval' 'unsafe-inline' https: http:;report-uri https://csp.withgoogle.com/csp/gws/other-hp
P3P: CP="This is not a P3P policy! See g.co/p3phelp for more info."
Date: Sun, 04 Feb 2024 22:48:38 GMT
Server: gws
X-XSS-Protection: 0
X-Frame-Options: SAMEORIGIN
Transfer-Encoding: chunked
Expires: Sun, 04 Feb 2024 22:48:38 GMT
Cache-Control: private
Set-Cookie: 1P_JAR=2024-02-04-22; expires=Tue, 05-Mar-2024 22:48:38 GMT; path=/; domain=.google.com; Secure
Set-Cookie: AEC=Ae3NU9OU79owVo5-35hp-3J9oKndh-rfsAa-zY1qEjSYgasatkOvi9Cc_P8; expires=Fri, 02-Aug-2024 22:48:38 GMT; path=/; domain=.google.com; Secure; HttpOnly; SameSite=lax
Set-Cookie: NID=511=YT39CFpNgnJnUOS1bI2BM3RWycjJIJ9HfoYYzzj48nu9tym-Ja_uLvmg-vBYyJYoA7_BBrlj4nGil9o0WYbz2tfMwDVzpHXRVZTZx7P9yI-Ad9t8VgRB4wVS5nGEi9xZ20djOLr-bTWoIiJuXgUdKH30znpJXziu542KaHu0ZcQ; expires=Mon, 05-Aug-2024 22:48:38 GMT; path=/; domain=.google.com; HttpOnly

